drivers/sensor/max17055/max17055.h

```diff
@@ -9,16 +9,35 @@
 
 /* Register addresses */
 enum {
-	REP_CAP		= 0x5,
-	REP_SOC		= 0x6,
-	INT_TEMP	= 0x8,
-	VCELL		= 0x9,
-	AVG_CURRENT	= 0xb,
-	FULL_CAP_REP	= 0x10,
-	TTE		= 0x11,
-	CYCLES		= 0x17,
-	DESIGN_CAP	= 0x18,
-	TTF		= 0x20,
+	STATUS          = 0x0,
+	REP_CAP         = 0x5,
+	REP_SOC         = 0x6,
+	INT_TEMP        = 0x8,
+	VCELL           = 0x9,
+	AVG_CURRENT     = 0xb,
+	FULL_CAP_REP    = 0x10,
+	TTE             = 0x11,
+	ICHG_TERM       = 0x1e,
+	CYCLES          = 0x17,
+	DESIGN_CAP      = 0x18,
+	TTF             = 0x20,
+	V_EMPTY         = 0x3a,
+	FSTAT           = 0x3d,
+	D_QACC          = 0x45,
+	D_PACC          = 0x46,
+	SOFT_WAKEUP     = 0x60,
+	HIB_CFG         = 0xba,
+	MODEL_CFG       = 0xdb,
+};
+
+/* Masks */
+enum {
+	FSTAT_DNR               = 0x0001,
+	HIB_CFG_CLEAR           = 0x0000,
+	MODELCFG_REFRESH        = 0x8000,
+	SOFT_WAKEUP_CLEAR       = 0x0000,
+	SOFT_WAKEUP_WAKEUP      = 0x0090,
+	STATUS_POR              = 0x0002,
```

Comment:
Is this change logically part of the same commit?

Reply:
See comment below.